packages/adena-extension/src/resources/chains/chains.json

```diff
@@ -1,17 +1,17 @@
 [
   {
-    "id": "test4",
+    "id": "test5",
     "default": true,
     "main": true,
-    "chainId": "test4",
+    "chainId": "test5",
     "chainName": "GNO.LAND",
-    "networkId": "test4",
-    "networkName": "Testnet 4",
+    "networkId": "test5",
+    "networkName": "Testnet 5",
     "addressPrefix": "g",
-    "rpcUrl": "https://rpc.test4.gno.land:443",
-    "indexerUrl": "https://test4.indexer.onbloc.xyz",
-    "gnoUrl": "https://test4.gno.land",
-    "apiUrl": "https://test4.api.onbloc.xyz",
+    "rpcUrl": "https://rpc.test5.gno.land:443",
```

Comment:
Let's use `https://test5.onbloc.xyz/` for now.

Reply:
I updated this commit. (https://github.com/onbloc/adena-wallet/pull/639/commits/de111727874c7f5c52dfb82462370b39e7fddf8e)